src/Okta.Core.Client/RegexUtils.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Globalization;
+
+namespace Okta.Core
+{
+    /// <summary>
+    /// Regular Expressions utilities
+    /// </summary>
+    public class RegexUtilities
```

Comment:
Thinking we should remove this class. It is only used for unit tests and doesn't need to be in the core project.